Runtime/Scripts/RGBotLocalRuntime/RGBotAssetsManager.cs

```diff
@@ -58,13 +58,8 @@ public void RefreshAvailableBots()
             foreach (var botGuid in botGuids)
             {
                 var botAssetPath = AssetDatabase.GUIDToAssetPath(botGuid);
-                var botDirectory = botAssetPath;
-                var lastIndex = botAssetPath.LastIndexOf(Path.DirectorySeparatorChar);
-                
-                if (lastIndex != -1) 
-                {
-                    botDirectory = botAssetPath.Substring(0, lastIndex);        
-                }
+                // unity assets always use '/' regardless of Operating System
+                var botDirectory = botAssetPath.Substring(0, botAssetPath.LastIndexOf('/'));
```

Comment:
Ah, so was the original issue that we were using the Path.DirectorySeparatorChar instead of just '/'?

Reply:
yep